docs/dev/run_the_etl.rst

```diff
@@ -11,24 +11,182 @@ just want to use already processed data.
 
 These instructions assume you have already gone through the :ref:`dev_setup`.
 
-There are two main scripts involved in the PUDL processing pipeline:
-
-1. ``ferc_to_sqlite`` :doc:`converts the FERC Form 1 DBF/XBRL files <clone_ferc1>` into
-   a single large `SQLite <https://sqlite.org>`__ database so that the data is easier to
-   extract, and so all of the raw FERC Form 1 data is available in a modern format. You
-   must run this script before you can run ``pudl_etl``.
-2. ``pudl_etl`` coordinates the "Extract, Transform, Load" process that processes
+Dagster
+-------
+PUDL uses `Dagster <https://dagster.io/>`__ to orchestrate data pipelines. Dagster
+makes it easy to manage data dependences, parallelize processes, cache results
+and handle IO. If you are planning on contributing to PUDL, it is recommended you
+read through the `Dagster Docs <https://docs.dagster.io/getting-started>`__ to
+familiarize yourself with the tool's main concepts.
+
+There are a handful of Dagster concepts worth understanding prior
+to interacting with the PUDL data processing pipeline:
+
+**Dagit:**
+`Dagit <https://docs.dagster.io/concepts/dagit/dagit>`__ is the Dagster
+UI for monitoring and executing ETL runs.
+
+**Software Defined Assets (SDAs):**
+
+    *An asset is an object in persistent storage, such as a table, file, or
+    persisted machine learning model. A software-defined asset is a Dagster object that
+    couples an asset to the function and upstream assets that are used to produce
+    its contents.*
```

Comment:
Are these sections in quotes because they are directly copied from a source? Asking because it seems weird that some of these have them and some of them don't.

Reply:
Yeah, the quotes are copied from the docs. I didn't use them for some concepts because I thought I could write a more concise/relevant description of the concept.